### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by urschrei**

### CODE REVIEW

**Comment by michaelkirk** [comment]
The current .travis-ci also makes sure that the `rstar-benches` crate builds, can you add that?

```
cargo build -p rstar-benches && cargo test --all-features -p rstar
```

And I suppose delete the .travis.yml

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  rstar:
+    name: rstar
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    defaults:
+      run:
+        working-directory: rstar
+    strategy:
+      matrix:
+        container_image: ["georust/geo-ci:rust-1.49", "georust/geo-ci:rust-1.50"]
```
Comment (michaelkirk) [inline .github/workflows/test.yml:33]: https://github.com/georust/docker-images/pull/12 😇

Comment (urschrei) [reply .github/workflows/test.yml:33]: Ha!

**Review by michaelkirk — APPROVED** [review]
LGTM!

**Comment by urschrei** [comment]
bors r=michaelkirk

**Pull request merged**